blackbox/docs/admin/system-information.rst

```diff
@@ -1042,6 +1042,44 @@ Every request that queries data or manipulates data is considered a "job" if it
 is a valid query. Requests that are not valid queries (for example, a request
 that tries to query a non-existent table) will not show up as jobs.
 
+Jobs Metrics
+------------
+
+The ``sys.jobs_metrics`` table provides an overview of the duration latencies
```

Comment:
Think this should be either "latency" or "duration". "duration latency" sounds wrong to me.